src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -239,8 +244,19 @@ class Queue[T <: Data](val gen: T,
     maybe_full := do_enq
   }
 
+  if (hasFlush) { //flushable queue specific logic
```

Comment:
can we make this comment more specific, like
```suggestion
  if (hasFlush) {
  // when flush is on, empty the queue
```